e2e/features/account/AddAccount.feature

```diff
@@ -23,6 +23,7 @@ Feature: AddAccount
         Given I click on a button with text "Go back"
```

Comment:
NRTTL
The charts on http://0.0.0.0:8080/#/transactions constantly refreshes and never becomes still

Reply:
As well as other strange behaviour when page refresh on most pages. 

https://github.com/LiskHQ/lisk-desktop/assets/8784876/7c372e15-530d-4efa-91c8-a38f87a67016